bundles/framework/divmanazer/component/SelectList.js

```diff
@@ -26,8 +27,18 @@ Oskari.clazz.define('Oskari.userinterface.component.SelectList', function (id) {
      * @param {Object} options
      * @return {jQuery Element} a list with chosen applied
      */
-    create: function (data, options) {
-        options = options || this.defaultOptions;
+    create: function (data, modifiedOptions) {
+        var me = this;
+        var getOptions = function (options) {
+            // extend default options to create new object where we concat passed values
+            var combinedOptions = jQuery.extend(true, {}, me.defaultOptions);
```

Comment:
You can pass options as fourth parameter so you won't have to do the manual copy loop.